index.html

```diff
@@ -4,7 +4,9 @@
     <meta charset="utf-8">
     <meta name="description" content="Labyrinth | Game ">
     <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no">
+    <meta name="google-signin-client_id" content="<use your own API key and remove this rotated triangle brackets while filling api>">
```

Comment:
are you sure that this is what must go in here ?

Reply:
Yup I can't deploy my api key and fill api key after removing "<>" this

Reply:
> Good work. Can you try to squash the commits into one?

okay ill make another pull request